governance/polls/templates/MIPs/Budget Ratification Poll - $mip_code - $date_MONTH_DD,_YYYY.md

```diff
@@ -57,7 +57,7 @@ This poll implements a **Minimum Positive Participation** value. The Minimum Pos
 
 **If the votes for Option 1 or Option 2 achieve a majority AND the votes for the winning option exceed 10,000 MKR, then the following actions will be taken:**
```

Comment:
I think this needs more changes. I'm not sure if the positive participation threshold is still a thing.

Reply:
I've added a parameter to check if this is a thing or not but have left the text in, just in case it still is.